cvat-core/src/server-proxy.ts

```diff
@@ -713,33 +757,7 @@ async function createProject(projectSpec: SerializedProject): Promise<Serialized
     }
 }
 
-async function getTasks(filter: TasksFilter = {}): Promise<SerializedTask[] & { count: number }> {
-    const { backendAPI } = config;
-    let response = null;
-    try {
-        if ('id' in filter) {
-            response = await Axios.get(`${backendAPI}/tasks/${filter.id}`);
-            const results = [response.data];
-            Object.defineProperty(results, 'count', {
-                value: 1,
-            });
-
-            return results as SerializedTask[] & { count: number };
-        }
-
-        response = await Axios.get(`${backendAPI}/tasks`, {
-            params: {
-                ...filter,
-                page_size: 10,
-            },
-        });
-    } catch (errorData) {
-        throw generateError(errorData);
-    }
-
-    response.data.results.count = response.data.count;
-    return response.data.results;
-}
+const getTasks = fetchAggregatedResource<TasksFilter, SerializedTask>(`${config.backendAPI}/tasks`, { page_size: 10 });
```

Comment:
`page_size` is a part of `TasksFilter`
Thus, it seems this setup filter will not have effect.

It is already in develop, I just emphasized it